docs/contributing/code-of-conduct.md

```diff
@@ -1,50 +1,50 @@
 
-## Our Responsibilities
+## Trách nhiệm của chúng tôi
 
-Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+Người duy trì dự án có nhiệm vụ nêu rõ mức tiêu chuẩn của hành vi của nhóm và thực thi một cách minh khai, công bằng và hợp lý cho những hành vi không đủ tiêu chuẩn.
 
-Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this code of conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+Người duy trì dự án có quyền và nghĩa vụ để xoá, sửa và không đồng ý các lời bình luận, commits, code, sửa wiki, issues, và các đóng góp khác mà không theo đủ tiêu chuẩn, và có thể tạm thời hoặc vĩnh viễn nhưng người đóng góp có hành vi không đàng hoàng, hăm doạ, súc phạm và tổn thương người khác.
 
-## Scope
+## Phạm vi của chúng tôi
 
-This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+Quy tắc ứng xử này áp dụng đối với cả trong không gian dự án và không gian công cộng khi cá nhân làm đại diện cho dự án hoặc cộng đồng. Ví dụ về đại diện cho một dự án hoặc một cộng đồng bao gồm việc sử dụng địa chỉ email chính thức của dự án, đăng tin qua tài khoản mạng xã hội chính thức hoặc đóng vai trò đại diện được chỉ định tại một sự kiện trực tuyến hoặc gặp gỡ trực tiếp. Đại diện của một dự án có thể được xác định và làm rõ hơn bởi những người duy trì dự án.
 
-## Enforcement
+## Quyền Thực Thi
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Các trường hợp lạm dụng, quấy rối hoặc hành vi không thể chấp nhận được có thể được báo cáo bằng cách liên hệ với nhóm dự án tại [conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). Tất cả các khiếu nại sẽ được xem xét và điều tra và đưa ra phản hồi cần thiết và phù hợp với hoàn cảnh. Nhóm dự án có nghĩa vụ duy trì bảo mật liên quan đến báo cáo sự cố. Thông tin chi tiết về các chính sách thực thi cụ thể có thể được đăng riêng biệt.
```

Comment:
1. confidentiality with regard to the reporter of an incident: tình bảo mật liên quan đến người báo cáo sự việc